src/lecture_event/lectureEvent.model.ts

```diff
@@ -0,0 +1,47 @@
+import mongoose from 'mongoose';
+import { ILectureEvent } from './lectureEvent.types';
+
+const lectureEventSchema = new mongoose.Schema<ILectureEvent>({
+  startDateTime: {
+    type: Date,
+    required: true,
+  },
+  endDateTime: {
+    type: Date,
+    required: true,
+  },
+  title: {
+    type: String,
+    required: true,
+    maxlength: 200,
+    minlength: 1,
+  },
+  groups: {
+    type: [Number],
+    required: true,
+  },
+  subgroups: {
```

Comment:
I talked to nojus and for now because his scrapper gets only one number lets just leave it as a number, so dont change the type change the name to subgroup